File: src/main/java/leets/leenk/domain/leenk/application/exception/CannotKickSelfException.java
package leets.leenk.domain.leenk.application.exception;

import leets.leenk.global.common.exception.BaseException;

public class CannotKickSelfException extends BaseException {
    public CannotKickSelfException() {
        super(ErrorCode.LEENK_CANNOT_KICK_SELF);
    }
}

File: src/main/kotlin/leets/leenk/domain/leenk/application/exception/CannotKickSelfException.kt
package leets.leenk.domain.leenk.application.exception;

import leets.leenk.global.common.exception.BaseException;

public class CannotKickSelfException extends BaseException {
    public CannotKickSelfException() {
        super(LeenkErrorCode.LEENK_CANNOT_KICK_SELF);
    }
}
